src/clusterfuzz/_internal/tests/appengine/libs/issue_management/google_issue_tracker/google_issue_tracker_test.py

```diff
@@ -429,6 +429,111 @@ def test_update_issue(self):
         mock.call().execute(http=None, num_retries=3),
     ])
 
+  def test_update_issue_to_security(self):
+    """Test updating an existing issue."""
+    self.client.issues().get().execute.return_value = BASIC_ISSUE
+    self.client.issues().modify().execute.return_value = {
+        'issueId': '68828938',
+        'issueState': {
+            'componentId':
+                '9001',
+            'type':
+                'ASSIGNED',
+            'status':
+                'NEW',
+            'priority':
+                'P2',
+            'severity':
+                'S2',
+            'title':
+                'issue title2',
+            'access_limit': {
+                'access_level': issue_tracker.IssueAccessLevel.LIMIT_NONE
+            },
+            'reporter': {
+                'emailAddress': 'reporter@google.com',
+                'userGaiaStatus': 'ACTIVE',
+            },
+            'assignee': {
+                'emailAddress': 'assignee2@google.com',
+                'userGaiaStatus': 'ACTIVE',
+            },
+            'retention':
+                'COMPONENT_DEFAULT',
+            'ccs': [{
+                'emailAddress': 'cc@google.com',
+                'userGaiaStatus': 'ACTIVE'
+            },],
+            'hotlistIds': ['12345',],
+        },
+        'createdTime': '2019-06-25T01:29:30.021Z',
+        'modifiedTime': '2019-06-25T01:29:30.021Z',
+        'userData': {},
+        'accessLimit': {
+            'accessLevel': 'INTERNAL'
+        },
+        'lastModifier': {
+            'emailAddress': 'user1@google.com',
+            'userGaiaStatus': 'ACTIVE',
+        },
+    }
+    issue = self.issue_tracker.get_issue(68828938)
+
+    # Mimic issue_filer's action in setting up the issue
+    issue.apply_extension_fields(EXTENSION_FIELDS)
+
+    issue.reporter = 'reporter@google.com'
+    issue.assignee = 'assignee2@google.com'
+    issue.ccs.add('cc@google.com')
+    issue.components.add('9001')
+    issue.labels.add('12345')
+    issue.status = 'ASSIGNED'
+    issue.title = 'issue title2'
+    issue.save()
+    self.assertEqual(68828938, issue.id)
+    self.client.issues().modify.assert_has_calls([
+        mock.call(),
```

Comment:
It is not clear to me why we have an additional (empty) call here.

Reply:
Maybe use remote-pdb and check who is executing it twice?
https://pypi.org/project/remote-pdb/

Reply:
> It is not clear to me why we have an additional (empty) call here.

Looks like this is expected and is likely due to the mocking framework and happens in other places where we specify a `return_value`.
Eg: ` self.client.issues().issueUpdates().list().execute.mock_calls` also has `mock.calls()` in it's calls list. But we do not assert against it because it is not the call we are interested in.

If we did care about asserting against it then we would be doing a `self.assertEqual(mock.mock_calls, [...])` everywhere instead of `assert_has_calls`.

I went ahead and removed the `mock.call()` from here because:
1. It is not relevant to the assertion
2. To be consistent with the rest of the test file
3. Tests pass as expected without it

This PR should be good to go now.